src/Model/Photo.php

```diff
@@ -130,18 +130,21 @@ public static function getPhotoForUser($uid, $resourceid, $scale = 0, array $con
 	 */
 	public static function getPhoto($resourceid, $scale = 0)
 	{
-		$r = self::selectFirst(["uid"], ["resource-id" => $resourceid]);
+		$r = self::selectFirst(["uid","allow_cid","allow_gid","deny_cid","deny_gid"], ["resource-id" => $resourceid]);
```

Comment:
Code standards: please add a space after commas.

Reply:
Done with latest commit.  Apologies for my poor familiarity with the coding standards.